alma/alma.php

```diff
@@ -604,10 +624,12 @@ public function getContent()
      * @return bool
      *
      * @throws \Alma\PrestaShop\Exceptions\CompatibilityPsAccountException
+     * @throws \PrestaShop\PsAccountsInstaller\Installer\Exception\ModuleNotInstalledException
      */
     public function renderPSAccount()
     {
         $this->checkCompatibilityPSModule();
+        $this->checkPsAccountCompatibility();
```

Comment:
The method names are a bit confusing, maybe you should:
- Only call `$this->checkPsAccountCompatibility();` here
- Rename `checkCompatibilityPSModule` to `checkPSAccountsPresence` or something similar
- Call it from `checkPsAccountCompatibility` as a first check

Reply:
@olance I agree with this. I updated it